.github/workflows/main.yml

```diff
@@ -152,6 +152,15 @@ jobs:
         with:
           package-import-name: "columbo"
 
+  validate-doc-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      
+      - name: Validate docs
+        run: ./docker/validate_docs.sh
+
```

Comment:
This runs the `validate_docs.sh` script in the CI process to make sure doc examples are valid.